### PR TITLE
Bear update, part 2

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

* Adds the black, brown (as a proper mob now), polars bear and combat bear (icon added to Hudson bear). Bear pelt hats included.
* The space bear no longer changes icons depending if you are on space or not.
* Added a polar bear to the lavaland_biodome_winter.
* Changed the type of bear that would spawn on a few other places.

* The polar bear pelt hat is no longer a fluff item.

Some sprites by @ATP-Engineer 
## Why It's Good For The Game

The brown bear was only an icon used for hallucinations, now hes a proper mob that can be used in other places.
More variation to the bears one can find is good, the pelt hats are also pretty cool and would be an desirable things for players to collect.

The Hudson bear and space bear could probably be added as enemies from space with the explorer redux PR that is currently open.
## Images of changes

bearsss
pelts
onmob

## Testing

- See images above
## Changelog
:cl: Henri215, ATP-Engineer
imageadd: Hudson, the bear now has a unique icon
add: Added the polar, black and brown bears into the game, pelt hats included
del: Space bears no longer change icons depending if they are on space or not
/:cl:

I could use suggestions for cool brown/black bear descriptions, maybe their pelts too.

### CODE REVIEW

**Comment by ATP-Engineer** [comment]
will the polar bear attack the arctic guys in the biodome?

**Review by S34NW — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 1]
```diff
@@ -36,23 +36,41 @@
 	gold_core_spawnable = HOSTILE_SPAWN
 	footstep_type = FOOTSTEP_MOB_CLAW
 
+/mob/living/simple_animal/hostile/bear/Process_Spacemove(movement_dir = 0)
+	return 1	//No drifting in space for space bears!
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/hostile/bear.dm:40]:
```suggestion
	return TRUE	//No drifting in space for space bears!
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Comment by Contrabang** [comment]
image

**Review by Contrabang — APPROVED** [review]
Lets do this

**Review by Contrabang — APPROVED** [review]

**Pull request merged**